EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh

```diff
@@ -0,0 +1,208 @@
+#ifndef EnergyScaleCorrection_class_hh
```

Comment:
Hi David, 

Sorry for the delay I had missed this comment. I'm reluctant to do this modification to a file of a package that I am not administrating. I would propose to merge this PR as it is, and to schedule a central check for wrong file names in CMSSW later on.